src/components/Landing/TrendingSection.tsx

```diff
@@ -0,0 +1,159 @@
+    paddingBottom: 0,
+  }
+
+  const headingStyles: CSSProperties = {
+    fontWeight: 600,
+    marginBottom: 15,
+    fontSize: 22,
+    marginTop: 20,
+    display: 'flex',
+    alignItems: 'center',
+    flexWrap: 'wrap',
+  }
+
+  const arrowIconSize = 14
+
+  const arrowIconStyle: CSSProperties = {
+    fontSize: arrowIconSize,
+    marginLeft: 5,
+  }
+
+  const dropdownSelectedBg = isDarkMode
+    ? colors.background.l0
+    : colors.background.brand.secondary
+
+  // Close dropdown when clicking anywhere in the window
+  window.addEventListener('click', () => setActiveKey(undefined), false)
+
+  const trendingWindowSelect = () => {
+    return (
+      <Collapse
+        className="trending-window-select"
+        style={{ border: 'none' }}
+        activeKey={activeKey}
+      >
+        <CollapsePanel
+          style={{
+            border: 'none',
+          }}
+          key={0}
+          showArrow={false}
+          header={
+            <div
+              onClick={e => {
+                setActiveKey(activeKey === 0 ? undefined : 0)
+                e.stopPropagation()
+              }}
+            >
+              {trendingWindow} days
+              {activeKey === 0 ? (
+                <UpOutlined style={arrowIconStyle} />
+              ) : (
+                <DownOutlined style={arrowIconStyle} />
+              )}
+            </div>
+          }
+        >
+          <div
+            onClick={e => e.stopPropagation()}
+            style={{
+              background: isDarkMode
+                ? colors.background.l2
+                : colors.background.l0,
+            }}
+          >
+            <option
+              className="select-item"
+              style={
+                trendingWindow === 7
+                  ? { backgroundColor: dropdownSelectedBg }
+                  : {}
+              }
+              onClick={() => setTrendingWindow(7)}
+            >
+              7 days
+            </option>
+            <option
+              className="select-item"
+              style={
+                trendingWindow === 30
+                  ? { backgroundColor: dropdownSelectedBg }
+                  : {}
+              }
+              onClick={() => setTrendingWindow(30)}
+            >
+              30 days
+            </option>
+          </div>
+        </CollapsePanel>
+      </Collapse>
+    )
+  }
+
+  return (
+    <section style={trendingProjectsStyle}>
+      <Row style={{ maxWidth: 1200, margin: 'auto' }}>
+        <Col xs={0} lg={8}>
+          <img
+            className="hide-mobile hide-tablet"
+            style={{ height: 550, marginTop: 46 }}
+            src="/assets/green_orange.png"
+            alt="Green orange singing"
+          />
+        </Col>
+        <Col xs={24} lg={15}>
+          <div style={{ paddingBottom: 20 }}>
+            <h3 style={headingStyles}>
+              {/* Only way I could get the title wrapping properly on mobile was making each word an element  
```

Comment:
**question**: how are we wanting it to wrap? It would be great to avoid this somehow 😅

Reply:
Only way I could think to avoid this happening:

https://user-images.githubusercontent.com/96150256/152640442-ed49f80a-c850-4a1a-9403-d4e5f3e9b813.mp4

This is how it works wrapping each word in its own element:

https://user-images.githubusercontent.com/96150256/152640506-fd8b1632-07b3-4201-bd06-136eac01eb3b.mp4